tests/files/api_fixtures/collection_operations.yaml

```diff
@@ -146,6 +146,43 @@ auth:
       }
 
 gcs:
+  - path: /collections
```

Comment:
Yeah, just missed an `add`. It's kind of sparse though, if you have any other suggestions; it's a pretty basic "smoke test" as is.

Reply:
The biggest thing is to run the command at all, and you've got that. 😄 

I think you should also run it with each option provided (combinations if you want), and make sure the option appears in the request query string. If you look for tests which check `responses`, you'll find examples which inspect the "last request sent", and you can crib from those.

That would cover/catch the issue with the parameters which need to be in `query_params`. Plus, when we update the SDK and move those opts back "up to" the top level, it will be a measure of protection against typos and other "silly mistakes".